src/SDL_sound_internal.h

```diff
@@ -260,6 +260,7 @@ typedef struct __SOUND_SAMPLEINTERNAL__
     Uint32 buffer_size;
     void *decoder_private;
     Sint32 total_time;
+    Uint32 total_length;
```

Comment:
You add a new member to the struct here, but only init it in VORBIS_open. I think you're supposed to init it at least to some default value in each function that initializes this struct.